bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/OpenWebNetBindingConstants.java

```diff
@@ -76,9 +79,14 @@
     public static final Set<ThingTypeUID> AUTOMATION_SUPPORTED_THING_TYPES = new HashSet<>(
             Arrays.asList(THING_TYPE_ZB_AUTOMATION, THING_TYPE_BUS_AUTOMATION));
 
+    // ## Energy Management
```

Comment:
Same for below.
```suggestion
    // Energy Management
```

Reply:
All comments in other sections begin with ##.
line 71: // ## Generic
line 74: // ## Lighting
line 78: // ## Automation
...and so on.
Should I leave it as is or do you want me to modify them all?

Reply:
Seems a bit strange with the double pound. I leave it up to you if you change all or keep all.

Reply:
I prefere to keep them all.